3주차/BE/yesjjin99/src/services/post.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Posts } from '../entities/post.entity';
+
+@Injectable()
+export class PostService {
+  @InjectRepository(Posts)
+  private readonly postRepository: Repository<Posts>;
```

Comment:
```suggestion
  constructor(
    @InjectRepository(Posts)
      private readonly postRepository: Repository<Posts>;
  ) {}
```
앞으로도 이런 식으로 생성자 주입 형태로 바꿔서 사용해 주시는 것을 추천드립니다.